CMakeLists.txt

```diff
@@ -4,7 +4,18 @@ cmake_minimum_required(VERSION 3.4)
 
 include(ExternalProject)
 
-project( CCTag )
+project(CCTag VERSION 1.0.0 LANGUAGES C CXX)
```

Comment:
The only drawback of this approach (it seems to me) is that if for some reason, god forbid, one day we need to switch to a different building system it may be possible that we cannot replicate this and to have to add `version.hpp` to the source code.